src/Evolution/Systems/GeneralizedHarmonic/Tags.hpp

```diff
@@ -53,17 +53,66 @@ struct ConstraintGamma2 : db::SimpleTag {
   using type = Scalar<DataVector>;
   static std::string name() noexcept { return "ConstraintGamma2"; }
 };
+/*!
+ * \brief Gauge source function for the generalized harmonic system.
+ * \details In the generalized / damped harmonic gauge, unlike the simple
+ * harmonic gauge, the right hand side of the gauge equation
+ * \f$ \square x_a = H_a\f$ is sourced by non-vanishing functions. This variable
+ * stores those functions \f$ H_a\f$.
+ */
 template <size_t Dim, typename Frame>
 struct GaugeH : db::SimpleTag {
   using type = tnsr::a<DataVector, Dim, Frame>;
   static std::string name() noexcept { return "GaugeH"; }
 };
+
+/*!
+ * \brief Spacetime derivatives of the gauge source function for the
```

Comment:
Should these tags be in the `GeneralizedHarmonicGroup` like other tags in this file?

Reply:
yes, good catch. added.